scripts/openmc-make-depletion-chain-casl

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python3
+
+import glob
+import os
+from zipfile import ZipFile
+from collections import OrderedDict, defaultdict
+from io import StringIO
+from itertools import chain
+
+try:
+    import lxml.etree as ET
+    _have_lxml = True
+except ImportError:
+    import xml.etree.ElementTree as ET
+    _have_lxml = False
+
+import openmc.data
+import openmc.deplete
+from openmc._xml import clean_indentation
+from openmc.deplete.chain import _REACTIONS
+from openmc.deplete.nuclide import Nuclide, DecayTuple, ReactionTuple
+from openmc._utils import download
+
+from casl_chain import CASL_CHAIN
+
+URLS = [
+    'http://www.nndc.bnl.gov/endf/b7.1/zips/ENDF-B-VII.1-neutrons.zip',
+    'http://www.nndc.bnl.gov/endf/b7.1/zips/ENDF-B-VII.1-decay.zip',
+    'http://www.nndc.bnl.gov/endf/b7.1/zips/ENDF-B-VII.1-nfy.zip'
+]
+
+def main():
+    if os.path.isdir('./decay') and os.path.isdir('./nfy') and os.path.isdir('./neutrons'):
+        endf_dir = '.'
+    elif 'OPENMC_ENDF_DATA' in os.environ:
+        endf_dir = os.environ['OPENMC_ENDF_DATA']
+    else:
+        for url in URLS:
+            basename = download(url)
+            with ZipFile(basename, 'r') as zf:
+                print('Extracting {}...'.format(basename))
+                zf.extractall()
```

Comment:
Note: It looks like this loop is extracting all the zip files in `URLS` to the current directory.  If any of the zip files have files with the same name, or any existing files in the current directory share a name with a file in the zip file, they will overwrite each other.  

If each of the zip files contains a root folder with a different name, this isn't an issue.  However, if they do not, one way to avoid this would be to pass a value to the `path` parameter of `extractall(...)` [1]

[1] https://docs.python.org/2/library/zipfile.html#zipfile.ZipFile.extractall